osquery/tables/system/windows/processes.cpp

```diff
@@ -144,7 +144,11 @@ Status getProcList(std::set<long>& pids) {
   return Status(0, "Ok");
 }
 
-void genProcess(const WmiResultItem& result, QueryData& results_data) {
+void genProcess(
+    const WmiResultItem& result,
+    QueryData& results_data,
+    QueryContext& context,
+    std::map<std::int32_t, std::map<std::string, std::int64_t>> perfData) {
```

Comment:
Make this a `std::map<std::int32_t, std::map<std::string, std::int64_t>>&`, and I'd say also make it the second parameter of the call, as we tend to keep the context and result params at the end.